### PR TITLE
add black window

### PR DESCRIPTION
**Pull request by tkwang0530**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    }
+
+    printf("Cleaning up...\n");
+    SDL_DestroyRenderer(rend);
+    SDL_DestroyWindow(win);
+    SDL_Quit();
+
+    printf("Done.\n");
+    return 0;
+}
```
Comment (novayo) [inline main.cpp:67]: Need EOF.

File: main.cpp [hunk 2]
```diff
@@ -0,0 +1,67 @@
+#include <stdio.h>
+#include <SDL2/SDL.h>
+#include <SDL2/SDL_timer.h>
+
+int main(int argc, char *argv[])
```
Comment (novayo) [inline main.cpp:5]: Please stay with the same style.
```suggestion
int main(int argc, char *argv[]) {
```

Comment (tkwang0530) [reply main.cpp]: revised

File: main.cpp (hunk 1 above)
Comment (tkwang0530) [reply main.cpp]: ok

**Review by novayo — APPROVED** [review]
LGTM!

**Pull request merged**